.golangci.yml

```diff
@@ -30,6 +30,7 @@ linters:
     - gosimple
     - govet
     - ineffassign
+    - importas
```

Comment:
I really like this change so I pulled it out to #518 to reduce the changes in this PR unrelated to the functionality it's providing.

Reply:
yeah just added this because at some point a lot of the packages were referencing v1cluster1.Cluster